### DISCUSSION TITLE
Please add new ENV variables for the docker container

### DISCUSSION THREAD

**Discussion by hueppinr** [wallabag/wallabag#7503] [Ideas]
I've come accross the php memory issue as some other users.
I propose a ENV variable "php memory limit" where a user can specify a larger amount than the default of 128M.

As I have problems getting RabbittMQ running, i found, that the default host in the docker is "localhost". I havn't found a way to change that. a new ENV variable to change this to another RabbitMQ instance already on the same machine would make it easier to set up.

thanks for listening to me and for the great software you provide

Kind regards 
Ralph

**Comment by j0k3r** [comment]
They both exist:
- in the main repo (only to be used to test): https://github.com/wallabag/wallabag/blob/0affa3709868e41eb5f7f51679319c53d19b42be/docker/php/config/wallabag-php.ini#L1
- in the official docker image (on the 2.6 branch): https://github.com/wallabag/docker/commit/588c21b192d7dcb7950166c6152831530dc5027c
